pkg/config/config.go

```diff
@@ -32,6 +37,12 @@ func New() Config {
 		// these are used when the image stream does not exist and created for the first time by the operator
 		MDCImageStreamInitialImage:        getEnv("MDC_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/aerogear/mobile-developer-console:latest"),
 		OauthProxyImageStreamInitialImage: getEnv("OAUTH_PROXY_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/openshift/oauth-proxy:v1.1.0"),
+
+		// override the default links displayed in MDC for each of the mobile services
+		UnifiedPushDocumentationURL:        getEnv("UPS_DOCUMENTATION_URL", ""),
```

Comment:
The second parameter to `getEnv` is for specifying a default that can be used if the env var isn't set. Is there a default that could be specified here?

Reply:
My thought is that we should default to the aerogear docs. This would be the default in the case of the community using the operator. We can override those in other cases. cc @finp

Reply:
I don't understand the context here.

Reply:
@grdryn I left these blank because the default for each service is already set in the code. 

Example: https://github.com/aerogear/mobile-developer-console/blob/ca2f317336e891457c546e85ed30a9568460d2fe/src/models/mobileservices/pushvariantcr.js#L282

We could also add them as defaults here.

Reply:
Ah right, if they're there in the mdc already, that's enough for me :+1:

Reply:
@craicoverflow I can provide links in here for you. I need to add some for downstream in the installer so we can do the same for upstream.

Reply:
Am I to wait for the links before merging this?

Reply:
@craicoverflow it's up to you. The links are there now in the linked pr. For example it would be https://docs.aerogear.org/limited-availability/upstream/sync.html

Reply:
thanks @laurafitzgerald - I've added the default values here now.